libs/application/templates/parental-leave/src/fields/PeriodPercentage/PeriodPercentage.tsx

```diff
@@ -136,6 +141,17 @@ export const PeriodPercentage: FC<PeriodPercentageField> = ({
   const isUsingAllRemainingDays =
     canChooseRemainingDays && selectedValue === maxPercentageValue
 
+  const getRatioTitle = (application: Application) => {
+    const appAnswers = getApplicationAnswers(application.answers)
```

Comment:
Same here as above, you can change the assignment to only store the needed variable.